website/docs/examples/jest-24.md

```diff
@@ -0,0 +1,11 @@
+# Use with Jest 24
+
+This example demonstrates how to use `jest-preview` with `jest-24`. The main intention of this example is to make sure Jest Preview works with Jest 24
+
+## Setup jest with vite
+
+If you haven't set up jest with vite yet, you can see how to do it at [Setup Jest with Vite](https://hung.dev/jest-vite) ([repo](https://github.com/nvh95/jest-with-vite)).
+
+## Installation and Usage
+
+Please refer to [Installation](../getting-started/installation.md) and [Usage](../getting-started/usage.md).
```

Comment:
Actually, we don't need to add every example to the docs site. 
The idea of `examples` folder I learned from Next.js (https://github.com/vercel/next.js/tree/canary/examples)
They attracts contributors to contribute by `examples`. And it's very easy for anyone to find their corresponding technology in `examples` folder.

It will be a ridiculous huge folder in the future. So I think we don't want to put every `example` to www.jest-preview.com

Reply:
For example, `jest-24` serve a little bit like a integration, more than an `example`.

Reply:
Okay, before we move to other comments and fixes, we need some sort of mapping of which examples we want to include, so then I can filter them out and work from there